integration/spring-pulsar/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot'
+    id 'org.springframework.aot.smoke-test'
+    id 'org.graalvm.buildtools.native'
+}
+
+repositories {
+    // Need this repo for the reactive pulsar client library.
+    maven {
+        url 'https://repository.apache.org/content/repositories/snapshots'
+    }
+}
+
+dependencies {
+    implementation(platform(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES))
+    implementation("org.springframework.pulsar:spring-pulsar-spring-boot-starter:0.1.0-SNAPSHOT")
+    implementation "org.apache.pulsar:pulsar-client-reactive-adapter:0.1.0-SNAPSHOT"
```

Comment:
If we are bringing in the reactive stuff we can replace these w/ the reactive pulsar starter as it includes both.

Reply:
That being said, is the plan to have a follow on PR that includes reactive support? If we do that then let's just include the imperative starter for now and remove this dep and also the repository from the build.gradle. Wdyt?

Reply:
I needed the reactive dependency because otherwise, the `nativeCompile` wasn't working.